packages/@webex/plugin-cc/test/unit/spec/cc.ts

```diff
@@ -1,12 +1,18 @@
 import 'jsdom-global/register';
 import {LoginOption, StationLogoutResponse, WebexSDK} from '../../../src/types';
+import WebSocketManager from '../../../src/services/core/WebSocket/WebSocketManager';
```

Comment:
Tests here are mostly the same, just include some of the mocks and the blob mocking. All seem to run properly and are passed.